blockchain/blockchain-wallet-v4-frontend
# THONG NGO

## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

[THONG NGO ](https://github.com/blockchain/blockchain-wallet-v4-frontend/compare/philip/wal-110-pending-transcactions-coin-view-wallet...analytics/hotfixes-violations-v1)
